packages/plexus/demo/src/index.tsx

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { render } from 'react-dom';
+
+import largeDg, { getNodeLabel as getLargeNodeLabel } from './data-large';
+import { edges as dagEdges, vertices as dagVertices } from './data-dag';
+import { colored as colorData, getColorNodeLabel, setOnColorEdge, setOnColorNode } from './data-small';
+import { DirectedGraph, LayoutManager } from '../../src';
+import { TVertex } from '../../src/types';
+
+// const { TVertex } = types;
+
+import './index.css';
+
+const { classNameIsSmall } = DirectedGraph.propsFactories;
+
+const addAnAttr = () => ({ 'data-rando': Math.random() });
+const setNodeClassName = (vertex: TVertex) => ({
```

Comment:
might as well call it `setOnNode` since that's how it's used and it does more than define the class name (unless the `onClick: () => console.log(...)` is meant to be removed)